napari/_qt/dialogs/screenshot_dialog.py

```diff
@@ -0,0 +1,47 @@
+import os
+from pathlib import Path
+from typing import Any, Callable
+
+from qtpy.QtWidgets import QFileDialog, QMessageBox
+
+
+class ScreenshotDialog(QFileDialog):
```

Comment:
Would be great to get a doc string on this class and the input parameters

Reply:
added